projects/gloo/cli/pkg/cmd/options/options.go

```diff
@@ -100,9 +100,32 @@ type Upgrade struct {
 }
 
 type Get struct {
+	Proxy    GetProxy
 	Selector InputMapStringString
 }
 
+type GetProxy struct {
+	// Name is not defined in this API, because all Get APIs allow you to specify a name of the object
+	// as the second argument: `glooctl get proxy named-proxy`
+
+	// Selector is not defined in this API, because the Get APIs allow you to specify a Selector already
+
+	// Namespace represents the namespace to search for the Proxy.
+	// An empty string is interpreted as all namespaces
+	Namespace string
+
+	// K8sGatewaySource is true if the GetProxy request should only consider Proxy resources
+	// that were generated by the K8s Gateway Translation
+	K8sGatewaySource bool
+
+	// EdgeGatewaySource is true if the GetProxy request should only consider Proxy resources
+	// that were generated by the Edge Gateway Translation
+	EdgeGatewaySource bool
```

Comment:
```suggestion
	// K8sGatewaySource is true if the GetProxy request should consider Proxy resources
	// that were generated by the K8s Gateway Translation
	K8sGatewaySource bool

	// EdgeGatewaySource is true if the GetProxy request should consider Proxy resources
	// that were generated by the Edge Gateway Translation
	EdgeGatewaySource bool
```
according to the implementation, both flags can be true at the same time, so probably more accurate to remove the word `only` in the 2 descriptions above